examples/inbox-multichain/pages/index.tsx

```diff
@@ -69,11 +78,29 @@ export default function Home(): JSX.Element {
   const { connection: solanaConnection } = useSolanaConnection();
   const solanaWallet = useSolanaWallet();
   const aptosWallet = useAptosWallet();
+  const { address: evmWallet, isConnected: evmWalletConnected } = useAccount();
 
   const [dialectSolanaWalletAdapter, setDialectSolanaWalletAdapter] =
     useState<DialectSolanaWalletAdapter | null>(null);
   const [dialectAptosWalletAdapter, setDialectAptosWalletAdapter] =
     useState<DialectAptosWalletAdapter | null>(null);
+  const [dialectEvmWalletAdapter, setDialectEvmWalletAdapter] =
+    useState<DialectEvmWalletAdapter | null>(null);
+
+  useEffect(() => {
+    if (!evmWalletConnected || !evmWallet) {
+      setDialectEvmWalletAdapter(null);
+    } else {
```

Comment:
wouldn't it cause problems when other chains connected?

Reply:
no, it's shouldn't